core/feconf.py

```diff
@@ -102,11 +102,9 @@
 RULES_DESCRIPTIONS_FILE_PATH = os.path.join(
     os.getcwd(), 'extensions', 'interactions', 'rule_templates.json')
 HTML_FIELD_TYPES_TO_RULE_SPECS_FILE_PATH = os.path.join(
-    os.getcwd(), 'extensions', 'interactions',
-    'html_field_types_to_rule_specs.json')
+    'interactions', 'html_field_types_to_rule_specs.json')
```

Comment:
I am a little worried about this, since we've always interpreted filepaths here to be absolute paths. I suppose it's fine to drop os.getcwd() but could we keep 'extensions' within this? It might also be worth renaming to *_RELATIVE_FILEPATH or similar.

Reply:
How's this?

Reply:
Yup I think it's OK, thanks!